spec/controllers/api/assignment_repos_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe API::AssignmentReposController, type: :controller do
+  let(:organization)      { classroom_org                                                           }
+  let(:user)              { classroom_teacher                                                       }
```

Comment:
This is some interesting style 🤔 
Nitpick: How about this?
```ruby
let(:organization) { classroom_org }
let(:user)         { classroom_teacher }
let(:assignment)   { create(:assignment, organization: organization, title: "Learn Clojure") }
```
I feel this reads slightly more effortlessly.

Reply:
Good point, fixed 👍